pkg/insights/resyncer.go

```diff
@@ -102,6 +102,10 @@ func (p *resyncer) Start(stop <-chan struct{}) error {
 			// because that's how we find online/offline Dataplane's status
 			continue
 		}
+		if resourceChanged.Type == core_mesh.MeshType || resourceChanged.Type == core_mesh.MeshInsightType {
```

Comment:
But there is a check in the line 97:
```go
if !meshScoped(resourceChanged.Type) {
    continue
}
```

Reply:
you are right. I don't know how did it work then...